app/models/beer.rb

```diff
@@ -7,7 +7,7 @@ class Beer < ActiveRecord::Base
   has_many :categories, through: :beer_categories
 
   validates :name, presence: true
-  validates :state, presence: true
+  validates :state, :inclusion => {:in => [true, false]}
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Space inside { missing.<br>Space inside } missing.